site-packages/integralstor/update_system_date_time.py

```diff
@@ -0,0 +1,28 @@
+from integralstor_utils import command
+
+def update_system_date_time(date, time):
+    try:
+	if not time == None:
+            cmd = 'date -s %s' %time
+            output, err = command.get_command_output(cmd)
+            if err:
+                raise Exception(err)
+        if not date == None:
+            date = date.split("/")
```

Comment:
Dont split it into the same variable. Always use different variables to get new info. So you should call it date_components

Also, dont use variables named date and time as that clashes with the python packages by the same name.